xchange-stream-service-netty/src/main/java/info/bitrich/xchangestream/service/netty/NettyStreamingService.java

```diff
@@ -319,7 +324,10 @@ public Completable disconnect() {
   public String getSubscriptionUniqueId(String channelName, Object... args) {
     return channelName;
   }
-
+  protected void sendMessageRateLimiterAcquire() {
```

Comment:
I'm not clear on what this is doing.

Reply:
maybe needs a comment that this function should be overridden to apply a rate limiter